documentation/docs/25-build-and-deploy/50-adapter-static.md

```diff
@@ -70,7 +70,7 @@ The directory to write static assets (the contents of `static`, plus client-side
 
 ### fallback
 
-Specify a fallback page for [SPA mode](single-page-apps), e.g. `index.html` or `200.html` or `404.html`.
+Specify a fallback page to generate for [SPA mode](single-page-apps) — e.g. `index.html` or `200.html` or `404.html`.
```

Comment:
```suggestion
Specify the name of the fallback page that SvelteKit should generate in [SPA mode](single-page-apps). For example: `index.html`, `200.html`, or `404.html`.
```

Reply:
This sounds like you have to enable SPA mode for the fallback to be generated, but in actuality generating the fallback it how you opt in to SPA mode.

Reply:
```suggestion
To create a [Single Page App](single-page-apps) you need to specify the name of the entry point generated by SvelteKit. This is commonly `index.html` or `200.html`, but can vary depending on your deployment platform.
```

What about something like this?

Reply:
looks good to me. I also added a few more tweaks if you'd like to take another look